src/libs/ValidationUtils.ts

```diff
@@ -478,6 +478,11 @@ function isExistingTaxName(taxName: string, taxRates: TaxRates): boolean {
     return !!Object.values(taxRates).find((taxRate) => taxRate.name === trimmedTaxName);
 }
 
+function isExistingTaxCode(taxCode: string, taxRates: TaxRates): boolean {
+    const trimmedTaxCode = taxCode.trim();
+    return !!Object.values(taxRates).find((taxRate) => taxRate.code === trimmedTaxCode);
```

Comment:
I think we might need to update this - cc @Gonals  - the code is (unfortunately) not stored this way. Instead:

<img width="720" alt="image" src="https://github.com/user-attachments/assets/b7c41375-de8b-46eb-a72d-9cb22f8ab20d">

the "id_TAX_EXEMPT" and "id_TAX_RATE_1" in this example are the tax codes. So we need to loop through `taxRates.taxes` and check if the key for the tax equals `trimmedTaxCode`

Reply:
nice catch! resolved